### DISCUSSION TITLE
[Beginner question] Plugin migrations do not run

### DISCUSSION THREAD

**Discussion by kff5000** [wintercms/winter#584] [Q&A]
Hi, I just started with Winter - and I'm not really a developer - but know, that winter is better then WP ;-) . So bear with me ;-)

I installed Winter via composer. Then artisan winter:install. Everything fine.

Then I did `composer require winter/wn-redirect-plugin` to install your redirect plugin. It appears in the backend menu. And I can see its configuration in the system settings.

But when I click it on it from the menu, I get a server error telling me that table `winter_redirect_categories` doesn't exist.

So I ran `php artisan winter:up` . But for Winter.Redirect it tells me: `- nothing to update` 

Also `php artisan plugin:refresh Winter.Redirect` doesn't help

And `php artisan plugin:list` doesn't show Winter.Redirect at all.

I'm surely missing something(?)

Thanks in advance,
Chris

**Comment by bennothommo** [comment]
@kff5000 are you able to confirm that the table doesn't exist in your database?

If you go to the **Plugins & Updates** area in the Settings, does it show the plugin as available and enabled?

**Reply by kff5000** [reply]
Yes: `No winter_redirect_`-table in the database. Only system tables, backend tables etc.

And no: It doesn't appear under Plugins&Updates. There I only find Demo, Builder and Drivers-Plugins

**Reply by bennothommo** [reply]
Are there records for Winter.Redirect within the `system_plugin_versions` and `system_plugin_history` tables?

**Reply by bennothommo** [reply]
And just to confirm, are the files installed in `plugins/winter/redirect`?

**Reply by kff5000** [reply]
No mentions of Winter.Redirect in 'system_plugin_versions' and 'system_plugin_history'. (only demo, builder, drivers)

Yes, the files exist in that location. Including the 'updates'-folder with migration-files inside

**Reply by bennothommo** [reply]
Which version of Winter CMS did you install?

**Reply by damsfx** [reply]
@kff5000 did your `/plugin/winter/redirect/updates/version.yaml` file contains the `v` prefix for version number and you're running winter 1.1.8?  
If so, you need to remove them.  

See:  
- https://github.com/wintercms/wn-redirect-plugin/pull/3
- https://github.com/wintercms/wn-redirect-plugin/pull/4

**Reply by bennothommo** [reply]
Nice spot @damsfx, although I just tried it with Winter 1.1.8 and it worked myself. Might be because I'm using SQLite, which might not be so strict about the data types going in.

**Reply by kff5000** [reply]
removed the 'v's from version.yaml - problem solved 💯 

Thank you both for the incredibly fast support

**Reply by damsfx** [reply]
> Nice spot @damsfx, although I just tried it with Winter 1.1.8 and it worked myself. Might be because I'm using SQLite, which might not be so strict about the data types going in.

I thought the problem was with the new VersionManager of Winter 1.2 that was not backport to 1.1.8?  
https://github.com/wintercms/wn-redirect-plugin/pull/3#issuecomment-1146575908

**Reply by kff5000** [reply]
I have 1.1.8 installed

**Comment by WebVPF** [comment]
@kff5000 After installing the plugin, run the command in the console:

```bash
php artisan winter:up
```

https://wintercms.com/docs/console/setup-maintenance#winter-up
